examples/bevy_wgpu_render.rs

```diff
@@ -90,7 +90,7 @@ fn setup(
             }
         }
     }
-    let quad_count = 2;
+    let quad_count = 1;
```

Comment:
I've decided that  2 quads were showing off a property of the engine that's going to be modified, so I reduced the quad size in the example to one.